CHANGELOG

```diff
@@ -138,6 +138,7 @@
 * Spreedly: Scrub sensitive transaction data [abarrak] #2781
 * Stripe: Add `exchange_rate` parameter [WilsonChiang] #2796
 * Mundipagg: New Gateway Implementation [nfarve] #2791
+* Integrated the Bambora Pacifc gateway into Bambora North America gateway #2797
```

Comment:
please put the name of the gateway first followed by `:`

Reply:
Fixed.